.circleci/config.yml

```diff
@@ -228,9 +228,12 @@ jobs:
             ln -s $(pwd)/julia-*/bin/julia /usr/bin/julia
             # conda update -y conda > /logs/conda.txt 2>&1
             # Install ESMValTool in a new conda environment
-            conda create -y --name esmvaltool -c esmvalgroup -c conda-forge esmvaltool 'python=3.9'
+            conda install -y -c conda-forge mamba
+            conda create -y --name esmvaltool
```

Comment:
Why not use mamba for the environment creation as well?

Reply:
yeah good point dude! Cheers!